src/linear_algebra/dual.lean

```diff
@@ -886,6 +897,37 @@ begin
   exact finrank_range_add_finrank_ker f,
 end
 
+open linear_map
+/-- `T` is surjective if and only if `T.dual_map` is injective -/
```

Comment:
This docstring is now backwards. I also think you should use `f` not `T` throughout, to match the rest of the file.